pkg/operator/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -209,6 +216,17 @@ func (n *clusterManagerController) sync(ctx context.Context, controllerContext f
 	}
 	managementClient := n.operatorKubeClient // We assume that operator is always running on the management cluster.
 
+	// If the work driver is not kube, we need to sync the work driver config secret
+	if workDriver != operatorapiv1.WorkDriverTypeKube {
+		if err := helpers.SyncWorkConfigSecret(ctx, n.operatorKubeClient, clusterManagerNamespace); err != nil {
```

Comment:
I think there is already a sync secret func in library-go

Reply:
replaced this with SyncSecret from library-go.